packages/language-server/src/commads.ts

```diff
@@ -39,6 +41,7 @@ export function executeCommand(
       connection.workspace.applyEdit({
         documentChanges: change,
       });
+      track("MANIFEST_STABLE_ID", "multiple");
```

Comment:
and here

Reply:
I made my own `track` function with a different API.
See:
```typescript
type TrackEventsKeys = keyof typeof TRACK_EVENTS;

export function track(
  eventType: TrackEventsKeys,
  ...custom_events: string[]
): void {
  try {
    getSWA().track(TRACK_EVENTS[eventType], custom_events);
  } catch (e) {
    console.error(`failed usage analytics tracking: ${e.message}`);
  }
}

So it accepts the key for the track event, not the value as the first argument
```